### ISSUE TITLE
About datasets details for training

### ISSUE THREAD

**Issue by RammusLeo** [xljh0520/JOTR#3]
Hi! I am trying to train the model based on the released checkpoints released. I use `MuCo`, `Human36M` and `CrowdPose` for finetuning and the MPJPE goes about 84 on 3DPW test split, which is a few worse than the best results.

Interestingly, I found that in config files `main_train.yaml` and `fine_tune.yaml`, the **trainset_3d** contains only `Human36M` and **trainset_2d** contains only `CrowdPose`. However, the paper mentions that the dataset used in training includes `MuCo-3DHP` and `MSCOCO` besides `Human36M` and `CrowdPose`. So I just want to check what datasets are used in training and fine-tune parts? And how can I select the datasets to keep the performance? Thank very much!

**Comment by xljh0520** [comment]
Thank you for pointing out the issue.
1. For the dataset, we utilized the four datasets mentioned in the paper. It was an oversight that the current config only included two of them. Since loading MSCOCO and MuCo takes a long time, I temporarily removed them during debugging. However, I have now added them back.
2. For achieving better results through finetuning, there are several hyperparameters that can be adjusted, such as learning rate, batch size, anchour number in contrastive learning and pos-neg pairs in contrastive learning. However, due to limited GPU resources, I am unable to perform a grid search on all of these hyperparameters. If conditions permit, you can experiment with adjusting these hyperparameters.

**Comment by RammusLeo** [comment]
OK, I understand. Thank you for your quick and patient reply!
Best wishes!
